### ISSUE TITLE
XLS downloads report an error on open

### ISSUE THREAD

**Issue by JoeGermuska** [censusreporter/censusreporter#47]
"Excel could not open ... because some content is unreadable."

Excel 2011 for Mac:
![image](https://cloud.githubusercontent.com/assets/46313/4240129/7819cf38-39e7-11e4-904b-a1038dc87eee.png)

A [report in UserVoice](https://censusreporter.uservoice.com/admin/tickets/57/) indicated this also happens with MS Office Professional 2013.

**Comment by JoeGermuska** [comment]
Not gone. I tried with a download from http://censusreporter.org/data/table/?table=B01001&geo_ids=140|16000US1714000 

I opened it in Excel 365 and got the same error, but also access to the repair logs:

```
<?xml version="1.0" encoding="UTF-8"?>
<recoveryLog xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main">
   <logFileName>Repair Result to acs2013_5yr_B01001_14000US170314305001.xml</logFileName>
   <summary>Errors were detected in file '/Users/germuska/Downloads/acs2013_5yr_B01001_14000US17031430500/acs2013_5yr_B01001_14000US17031430500.xlsx'</summary>
   <repairedRecords summary="Following is a list of repairs:">
      <repairedRecord>Repaired Records: Worksheet properties from /xl/workbook.xml (Workbook)</repairedRecord>
      <repairedRecord>Repaired Records: Cell information from /xl/worksheets/sheet1.xml</repairedRecord>
   </repairedRecords>
</recoveryLog>
```

**Comment by JoeGermuska** [comment]
fixed in ff6c6e6, deployed
